DemoServer/Startup.cs

```diff
@@ -50,10 +51,9 @@ public void ConfigureServices(IServiceCollection services)
 
             services.AddSingleton<Settings>(settings);
 
-            var serviceProvider = services.BuildServiceProvider();
             services.AddMemoryCache();
 
-            services.AddMvc().SetCompatibilityVersion(CompatibilityVersion.Version_2_1);
+            services.AddMvc();
```

Comment:
Do we need `AddMvc()` while we have `AddControllers()` ? I mean, do we use `RazorPages` or classic MVC with `Razor Views`?